mits/templates/mits_applications/index.html

```diff
@@ -60,7 +60,7 @@ <h3>Manage Your Team</h3>
     </tr>
 {% endfor %}
 </table>
-{% if team.applicants|length < c.MITS_BADGES_PER_TEAM %}
+{% if team.can_add_badges %}
```

Comment:
This is a minor fix now that we have badge linking.  In the past we only allowed teams to add 3 badges.  But in theory some people may have already paid for a badge, so now we don't count those badges against their total.